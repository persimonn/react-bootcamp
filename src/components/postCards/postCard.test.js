import React from "react";
import toJson from "enzyme-to-json";
import { shallow } from "enzyme";
import PostCard, { sum } from "./index.js";

describe("testing postCard", () => {
  const wrapper = shallow(<PostCard />);

  it("sums to numbers", () => {
    expect(sum(3, 5)).toEqual(8);
  });

  it("renders properly", () => {
    expect(wrapper.length).toEqual(1);
  });

  it("renders snapshot", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
