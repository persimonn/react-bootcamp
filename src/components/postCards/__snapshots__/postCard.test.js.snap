// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing postCard renders snapshot 1`] = `
<div
  className="col-12 mt-5"
>
  <div
    className="card"
  >
    <img
      className="card-img"
    />
    <div
      className="card-body card-margin"
    >
      <h5
        className="card-title"
      />
      <p
        className="card-text"
      />
    </div>
  </div>
</div>
`;
